packages/expo-build-properties/src/pluginConfig.ts

```diff
@@ -338,6 +338,17 @@ export interface PluginConfigTypeIos {
    * and [Apple's documentation on Privacy manifest files](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files).
    */
   privacyManifestAggregationEnabled?: boolean;
+
+  /**
+   * Enable support for prebuilt React Native iOS dependencies (ReactNativeDependencies.xcframework).
```

Comment:
```suggestion
   * Enables support for prebuilt React Native iOS dependencies (`ReactNativeDependencies.xcframework`).
```

Let's use inline syntax highlight here.